src/pages/Root.tsx

```diff
@@ -0,0 +1,11 @@
+import Summary from '../components/main/Summary'
+import { motion } from 'framer-motion'
+import { pageTransitionOne } from '../components/Animation/page-animation'
+
+const Root = () => {
+    return <motion.div initial='out' animate='in' exit='out' variants={pageTransitionOne}>
```

Comment:
since we are calling this animation style everywhere, it is a prime example for a higher order component that should render its children

Reply:
ok done